CMakeLists.txt

```diff
@@ -33,9 +33,9 @@ endforeach()
 include(${CMAKE_CURRENT_SOURCE_DIR}/CMake/ITKInitializeCXXStandard.cmake)
 include(${CMAKE_CURRENT_SOURCE_DIR}/CMake/ITKInitializeBuildType.cmake)
 
-# ==== Define language standard configurations requiring at least c++14 standard
-if(CMAKE_CXX_STANDARD EQUAL "98" OR CMAKE_CXX_STANDARD LESS "14")
-   message(FATAL_ERROR "C++98 to C++11 are no longer supported in ITK version 5.3 and greater.")
+# ==== Define language standard configurations requiring at least c++17 standard
+if(CMAKE_CXX_STANDARD EQUAL "98" OR CMAKE_CXX_STANDARD LESS "17")
+   message(FATAL_ERROR "C++98 to C++14 are no longer supported in ITK version 5.3 and greater.")
```

Comment:
ITK version 5.3 -> ITK version 5.4

Reply:
@dzenanz Thanks, addressed with the most recent [force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/099c723c5d94bf12f2a188ad185042e3dbcc8067..0ec405cafff5ab1bfed0cb5c494755c151a3ca81) 😃